### ISSUE TITLE
NameError: global name 'MemoryException' is not defined

### ISSUE THREAD

**Issue by binjo** [bruce30262/TWindbg#2]
Hi, I got exception as follows:
```
Traceback (most recent call last):

  File "c:\pykdscript\TWindbg.py", line 86, in onExecutionStatusChange
    self.print_context()

  File "c:\pykdscript\TWindbg.py", line 95, in print_context
    self.print_stack()

  File "c:\pykdscript\TWindbg.py", line 166, in print_stack
    ptr_values = self.smart_dereference(cur_sp)

  File "c:\pykdscript\TWindbg.py", line 180, in smart_dereference
    except MemoryException: # no more dereference

NameError: global name 'MemoryException' is not defined
```

adding module name fixed it
```
diff --git a/TWindbg/TWindbg.py b/TWindbg/TWindbg.py
index daf8966..18be21f 100644
--- a/TWindbg/TWindbg.py
+++ b/TWindbg/TWindbg.py
@@ -177,7 +177,7 @@ class ContextHandler(pykd.eventHandler):
                 val = pykd.loadPtrs(addr, 1)[0]
                 ret.append(val)
                 addr = val
-            except MemoryException: # no more dereference
+            except pykd.MemoryException: # no more dereference
                 break
 
         return ret
```

**Comment by bruce30262** [comment]
@binjo thanks for reporting the issue
It's kind of weird though, since I have no problem using the extension without adding the module name.
I'll fix it later, thanks again !

**Comment by bruce30262** [comment]
Fixed in 15940c3965f4e6  
Feel free to reopen the issue if the error still occur
